app/views/comments/new.html.erb

```diff
@@ -1,5 +1,7 @@
 <div>
-   
+    <h2> Create Comment </h2>
+    <%= form_with model: @comment, url: user_post_comments_path, data: { turbo: false} do |form| %>
+        
         <div class="mb-3">
             <%= form.label :text, class: "form-label" %>
             <%= form.text_field :text, class: "form-control", id: "exampleFormControlInput1", placeholder: "Post Comment" %>

```

Comment:
- Hey, the User can't add a comment as there appears to be no form in your UI to integrate that. Also, one can't create a post as there's no form to add this. please, kindly ensure to fix this by adding a form to create a `Post`, add a `Comment`, and add a `Like`. These features are missing in your app. You should have a form like this:

![image](https://github.com/techEmprez/Blog_app/assets/59397159/c5e39854-b56f-4943-bd46-0c24a79f9378)

![image](https://github.com/techEmprez/Blog_app/assets/59397159/321eeb38-26e4-4771-9309-19bbef082143)
![image](https://github.com/techEmprez/Blog_app/assets/59397159/a7e77c79-b9be-49f5-82f5-fed352ac48a9)

Reply:
All done